tusker/config.py

```diff
@@ -46,6 +47,13 @@ def get(self, name, type, required=False, default=None):
             else:
                 return default
         value = self.data[name]
+        if isinstance(value, str) and re.match(r"^\${[a-zA-Z_]+}$", value):
+            env_variable = value[2:-1];
```

Comment:
With a replacement function that matches the entire `${...}` string you would not need to do any of this string slicing.